src/feature_alert/tooltip.es

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+import { invoke } from 'nebenan-helpers/lib/utils';
+
+import { useEscHandler, useMisClickHandler } from './hooks';
+import { getTriggerProps } from './utils';
+import {
+  POSITION_TOP,
+  POSITION_BOTTOM,
+  POSITION_LEFT,
+  POSITION_RIGHT,
+  TRIGGER_HOVER,
+  TRIGGER_CLICK,
+  TRIGGER_DELAYED,
+} from './constants';
+
+const DELAY_TIMEOUT = 1000 * 3;
+
+
+const FeatureAlertTooltip = (props) => {
+  const {
+    position,
+    trigger,
+    content,
+    children,
+    closeIcon,
+    defaultOpen,
+    onOpen,
+    onClose,
+    ...cleanProps
+  } = props;
+
+  const [isOpen, setOpen] = useState(false);
+  const ref = useRef(null);
+
+  // need to be able to only open once
+  const wasActive = useRef(null);
+  const isActive = useRef(false);
+
+  const handleOpen = (event) => {
+    if (event) event.stopPropagation();
+    if (wasActive.current) return;
+    wasActive.current = true;
+    isActive.current = true;
+    setOpen(true);
+    invoke(onOpen);
+  };
+
+  const handleClose = () => {
+    if (!isActive.current) return;
+    isActive.current = false;
+    setOpen(false);
+    invoke(onClose);
+  };
+
+  useEscHandler(handleClose);
+  useMisClickHandler(ref, handleClose);
```

Comment:
You are using props (`onClose` and `onOpen`) inside those hooks. Think you need to add them as dependencies to the useEffect calls.